test/e2e/density.go

```diff
@@ -379,14 +384,10 @@ var _ = Describe("Density", func() {
 				printLatencies(e2eLag, "worst e2e total latencies")
 
 				// Test whether e2e pod startup time is acceptable.
+				podStartupLatency := PodStartupLatency{Latency: extractLatencyMetrics(e2eLag)}
 				// TODO: Switch it to 5 seconds once we are sure our tests are passing.
 				podStartupThreshold := 8 * time.Second
-				e2ePodStartupTime50perc := e2eLag[len(e2eLag)/2].Latency
-				e2ePodStartupTime90perc := e2eLag[len(e2eLag)*9/10].Latency
-				e2ePodStartupTime99perc := e2eLag[len(e2eLag)*99/100].Latency
-				Expect(e2ePodStartupTime50perc < podStartupThreshold).To(Equal(true), "Too high pod startup time 50th percentile")
-				Expect(e2ePodStartupTime90perc < podStartupThreshold).To(Equal(true), "Too high pod startup time 90th percentile")
-				Expect(e2ePodStartupTime99perc < podStartupThreshold).To(Equal(true), "Too high pod startup time 99th percentile")
+				expectNoError(AcceptablePodStartupLatency(podStartupLatency, podStartupThreshold))
```

Comment:
To be honest I find this function name confusing. How about ```VerifyPodStartupLatency``` or ```CheckPodStartupLatency```? I think it will be more obvious that it may return error.